src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
- [ ] Unfortunately, your application has stopped compiling. This is probably because of naming errors. I suggest fixing these issues so the next reviewer can perform the review correctly. If you are not encountering any issues locally, is probably that your Git is out of sync with your files. My suggestion here is that you remove the project from your local machine and try cloning again from GitHub.

The compiling errors are probably the same ones that Netlify is encountering, hence, it could not deploy correctly your live version.

Image of the compiling errors:
![image](https://user-images.githubusercontent.com/79114233/145685801-c43357a2-fdd0-4da8-8235-4e9c3165fde1.png)

[OPTIONAL] I highly recommend you merge branches approved before opening pull request, that way each PR will only have their unique changes instead of the entire project. Which will result in better code reviews.

Reply:
Thank you for the review. Fixed changes